doc/radosgw/encryption.rst

```diff
@@ -37,10 +37,11 @@ or decrypt data.
 
 This is implemented in S3 according to the `Amazon SSE-KMS`_ specification.
 
-In principle, any key management service could be used here, but currently
-only integration with `Barbican`_ and `Vault`_ are implemented.
+In principle, any key management service could be used here.  Currently
+integration with `Barbican`_, `Vault`_, and `KMIP`_ are implemented.
 
-See `OpenStack Barbican Integration`_ and `HashiCorp Vault Integration`_.
+See `OpenStack Barbican Integration`_, `HashiCorp Vault Integration`_.
+and `KMIP Integration`_.
```

Comment:
comma instead of period between `HashiCorp Vault Integration. and KMIP Integration.`